backend/src/main/repositories/AccountRepository.ts

```diff
@@ -27,11 +30,8 @@ export default class AccountRepository implements CRUD {
   public delete = async (email: string): Promise<number> => {
     try {
       const deletedAccountStatus = await Account.destroy({
-        where: {
```

Comment:
In this file, I added some spaces so that the code is clearer. Did prettier change it here or did you do it yourself?

Reply:
I added the space back